### DISCUSSION TITLE
How to permanently delete my deleted repos?

### DISCUSSION THREAD

**Discussion by pastrami1764** [community/community#42710] [Repositories] [answered]
### Select Topic Area

Question

### Body

I deleted a couple of repositories from my personal account and apparently I can still recover them using the instructions listed [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository).
I would like to permanently delete my deleted repositories. Is there a way I can do that? Reading through some tangentially related discussions it seems that repositories are permanently deleted after 90 days of deletion. Is that true and is there a way to expedite that process?
Thank you in advance.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hi @pastrami1764 

When the repository is deleted, it takes up to 1 hour for it to appear in this list of "Deleted Repositories", after 90 days in this list, it will be permanently deleted.

You can find detailed information in the link below

[https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository)
